src/topology/metric_space/hausdorff_distance.lean

```diff
@@ -835,16 +870,24 @@ is_closed.preimage continuous_inf_edist is_closed_Iic
 @[simp] lemma cthickening_empty (δ : ℝ) : cthickening δ (∅ : set α) = ∅ :=
 by simp only [cthickening, ennreal.of_real_ne_top, set_of_false, inf_edist_empty, top_le_iff]
 
+lemma cthickening_of_nonpos {δ : ℝ} (hδ : δ ≤ 0) (E : set α) :
```

Comment:
I added three of your lemmas in PR #10661 and possibly placed them in a different order than here. The diff in this PR shows only a diff to an earlier version. In this form, there may be a merge conflict with current `master`.

Reply:
Merged master.